server/models/video/video-channel.ts

```diff
@@ -151,16 +156,52 @@ export type SummaryOptions = {
       }
     ]
   },
-  [ScopeNames.WITH_VIDEOS]: {
+  [ScopeNames.WITH_ACTOR]: {
     include: [
-      VideoModel
+      ActorModel
     ]
   },
-  [ScopeNames.WITH_ACTOR]: {
+  [ScopeNames.WITH_VIDEOS]: {
     include: [
-      ActorModel
+      VideoModel
     ]
-  }
+  },
+  [ScopeNames.WITH_STATS]: (options: AvailableWithStatsOptions = { daysPrior: 30 }) => ({
```

Comment:
So we don't have security issues in the future, force `daysPrior` to be an int -> `daysPrior = parseInt(daysPrior + '', 10)`